ccmlib/scylla_node.py

```diff
@@ -1336,12 +1336,16 @@ def rollback(self, upgrade_to_version):
     def hostid(self, timeout=60, force_refresh=False):
         if self.node_hostid and not force_refresh:
             return self.node_hostid
-        node_address = self.address()
-        url = f"http://{node_address}:10000/storage_service/hostid/local"
-        response = requests.get(url=url)
-        if response.status_code == requests.codes.ok:
-            self.node_hostid = response.json()
-            return self.node_hostid
+        try:
+            node_address = self.address()
+            url = f"http://{node_address}:10000/storage_service/hostid/local"
+            response = requests.get(url=url)
+            if response.status_code == requests.codes.ok:
+                self.node_hostid = response.json()
+                return self.node_hostid
+        except Exception as e:
+            self.error(f"Failed to get hostid using {url}: {e}")
+            pass
```

Comment:
‎This "pass" isn't needed.

Reply:
True, but I find it clarifying, as it documents the fact that the error is ignored on purpose.
Otherwise, a naive reader might suspect there is a missing `raise`.

Reply:
dropped `pass` in next version